### ISSUE TITLE
Inconsistency in the annotations

### ISSUE THREAD

**Issue by UndecidedBoy** [bozcani/auairdataset#3]
Hello,
Thanks for making the dataset available!

I had a question about the JSON file containing the annotations.
I downloaded the annotations from the link provided in:
 - https://bozcani.github.io/auairdataset

But the annotations are not consistent from one frame to an other:
For example in this section:
 - **frame_n :** *frame_20190905091750_x_0004180.jpg*
![frame_n](https://user-images.githubusercontent.com/35936764/74929634-38dadf00-53dc-11ea-961f-75fca00403d4.png)

 - **frame_n+1**
![frame_n1](https://user-images.githubusercontent.com/35936764/74929616-2f517700-53dc-11ea-8699-babb79a8d601.png)

 - **frame_n+2**
![framen3](https://user-images.githubusercontent.com/35936764/74929664-485a2800-53dc-11ea-8125-c7578640b822.png)

 - **frame_n+3**
![framen4](https://user-images.githubusercontent.com/35936764/74929669-4abc8200-53dc-11ea-8fa2-785ce6c60df1.png)

Is there possibly a better annotation file?

**Comment by bozcani** [comment]
Thanks for opening this issue @UndecidedBoy

Currently, we only have the shared annotation file now. 

We have used crowdsource for labeling (Amazon Mechanical Turk). We tried our best to manually inspect samples. But there are overlook samples (~32K samples to inspect in total) that have mislabeled as you shared.

I am working on better annotations (consistent and tight boxes) now and planning to update the annotation file in April, hopefully.
